firmware/configs/sha2017_defconfig

```diff
@@ -806,7 +806,7 @@ CONFIG_MICROPY_USE_DISPLAY=
 CONFIG_MICROPY_USE_GPS=
 CONFIG_MICROPY_USE_ETHERNET=
 CONFIG_MICROPY_USE_MDNS=
-CONFIG_MICROPY_USE_REQUESTS=y
+CONFIG_MICROPY_USE_REQUESTS=n
```

Comment:
Can you remove this "n" and instead just clear the field?
Also: please separate these changes or rename your PR, pick one :D

Reply:
... that's weird, I based this on my other PR a while ago with these two commits, and would've expected it to remove them when that's merged. I'll just rebase. <_<